src/virtual-repeat.js

```diff
@@ -271,35 +271,39 @@ export class VirtualRepeat extends AbstractRepeater {
   _getMore(): void {
     if (this.isLastIndex || this._first === 0) {
       if (!this._calledGetMore) {
-        let getMoreFuncName = this.view(0).firstChild.getAttribute('virtual-repeat-next');
-        if (!getMoreFuncName) {
-          return;
-        }
-        let getMoreFuncParts = getMoreFuncName.split('.');
-        let getMoreFunc = this.scope.overrideContext.bindingContext || {};
-        for (let propertyIndex = 0; propertyIndex < getMoreFuncParts.length; propertyIndex++) {
-          getMoreFunc = getMoreFunc[getMoreFuncParts[propertyIndex]];
-          if (!getMoreFunc) {
-            throw new Error("Can't locate 'virtual-repeat-next' function");
-          }
-        }
         let executeGetMore = () => {
           this._calledGetMore = true;
-          if (getMoreFunc instanceof Promise) {
-            return getMoreFunc.then(() => {
-              this._calledGetMore = false; //Reset for the next time
-            });
-          } else if (typeof getMoreFunc === 'function') {
-            let result = getMoreFunc.bind(this.scope.overrideContext.bindingContext)(this._first, this._bottomBufferHeight === 0, this._isAtTop);
-            if (!(result instanceof Promise)) {
-              this._calledGetMore = false; //Reset for the next time
-            } else {
-              return result.then(() => {
+          let func = this.view(0).firstChild.au['virtual-repeat-next'].instruction.attributes['virtual-repeat-next'];
```

Comment:
We have already injected the DOM element, so can't we just use
`this.element.au['virtual-repeat-next'].instruction.attributes['virtual-repeat-next'];`